### PR TITLE
Couture v2

### PR DESCRIPTION
**Pull request by AdrianMirandaTailor**
---
Issue link: WILL BE INSERTED BY [boring-cyborg](https://github.com/kaxil/boring-cyborg)

Make sure to mark the boxes below before creating PR: [x]

- [ ] Description above provides context of the change
- [ ] Commit message/PR title starts with `[AIRFLOW-NNNN]`. AIRFLOW-NNNN = JIRA ID<sup>*</sup>
- [ ] Unit tests coverage for changes (not needed for documentation changes)
- [ ] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [ ] Relevant documentation is updated including usage instructions.
- [ ] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

<sup>*</sup> For document-only changes commit message can start with `[AIRFLOW-XXXX]`.

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: airflow/www_rbac/views.py [hunk 1]
```diff
@@ -3656,19 +3655,19 @@ def update_new_args_configs(self, config):
     def update_spark_conf(self, group):
         title = "Couture Spark Configuration"
 
+        
         config = CP.ConfigParser()
         config.optionxform = str
         conf_path = os.path.join(
             settings.HADOOP_CONFIGS_FOLDER, *[group, 'couture-spark.conf'])
-        setup_path = settings.SPARK_DEPENDENCIES_FOLDER
+        setup_path = os.path.join(settings.AIRFLOW_HOME, *[os.pardir, 'jars'])
```
Comment (Anubhav2701) [inline airflow/www_rbac/views.py]: setup_path = settings.SPARK_DEPENDENCIES_FOLDER

File: airflow/www_rbac/views.py [hunk 2]
```diff
@@ -3656,19 +3655,19 @@ def update_new_args_configs(self, config):
...
             settings.HADOOP_CONFIGS_FOLDER, *[group, 'keytab'])
         args, configs = self.init_args_configs(conf_path, config)
 
-        files = self.get_files(setup_path, [".jar"])
-        print(files, setup_path)
-        py_files = self.get_files(setup_path, [".py",".egg",".zip"])
-        kt_files = self.get_files(keytab_path)
+        files = self.get_files([".jar"], setup_path)
+        py_files = self.get_files([".py",".egg",".zip"], setup_path)
+        kt_files = self.get_files(["kt"],keytab_path)
```
Comment (Anubhav2701) [inline airflow/www_rbac/views.py]: Revert any of your current changes in SparkConfView

File: airflow/www_rbac/views.py [hunk 3]
```diff
@@ -3915,7 +3912,9 @@ def update_keytab(self, group):
     @has_access
     def download(self, group, filename):  # for downloading the file passed in the filename
         add_to_dir = os.path.join(settings.HADOOP_CONFIGS_FOLDER, *[group])
-        path_file = os.path.join(add_to_dir, filename)
+        keytab_path = os.path.join(add_to_dir, 'keytab/')
```
Comment (Anubhav2701) [inline airflow/www_rbac/views.py]: remove trailing /

File: airflow/www_rbac/views.py [hunk 4]
```diff
@@ -3915,7 +3912,9 @@ def update_keytab(self, group):
...
+        path_file = os.path.join(keytab_path, filename)
+        print(path_file)
```
Comment (Anubhav2701) [inline airflow/www_rbac/views.py]: remove print

Comment (AdrianMirandaTailor) [reply airflow/www_rbac/views.py]: done

File: airflow/www_rbac/views.py (hunk 1 above)
Comment (AdrianMirandaTailor) [reply airflow/www_rbac/views.py]: done

**Comment by Anubhav2701** [comment]
LGTM

**Pull request merged**